src/core/src/main/java/org/apache/jmeter/testelement/TestPlan.java

```diff
@@ -83,11 +86,23 @@ public void prepareForPreCompile()
 
     /**
      * Fetches the functional mode property
-     *
```

Comment:
If we want to keep the space to be visible, we should add something like `<br>` or `<p>`.

Reply:
`<br>` is wrong here.
The sentence should end with a period. See https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#descriptions

See

>  The Javadoc tool copies this first sentence to the appropriate member, class/interface or package summary